From: Boyuan Yang <byang@debian.org>
Date: Sat, 23 May 2020 19:54:52 -0400
Subject: CMakeLists: use GNUInstallDirs
Forwarded: not-needed

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ba692ff..87785ea 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 # Many projects still are stuck using CMake 2.8 is several places so it's good to provide backward support too. This is
 # specially true in old embedded systems (OpenWRT and friends) where CMake isn't necessarily upgraded.
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.0.2)
+
+include(GNUInstallDirs)
 
 if(POLICY CMP0048)
 	cmake_policy(SET CMP0048 NEW)
