From: Nicolas Mora
Date: Thu, 23 Jul 2020 10:31:12 -0400
Subject: config.h: remove macro PACKAGE_NAME
Forwarded: not-needed

---
--- a/cmake/config.h.in
+++ b/cmake/config.h.in
@@ -160,7 +160,9 @@
 #define PACKAGE_BUGREPORT "@JSON_C_BUGREPORT@"
 
 /* Define to the full name of this package. */
-#define PACKAGE_NAME "@PROJECT_NAME@"
+// Macro disabled to avoid collisions with other libraries
+// Lintian warning: package-name-defined-in-config-h
+//#define PACKAGE_NAME_ "@PROJECT_NAME@"
 
 /* Define to the full name and version of this package. */
 #define PACKAGE_STRING "@PROJECT_NAME@ @CPACK_PACKAGE_VERSION_MAJOR@.@CPACK_PACKAGE_VERSION_MINOR@.@CPACK_PACKAGE_VERSION_PATCH@"
