Description: Make the build reproducible
Author: Chris Lamb <lamby@debian.org>
Last-Update: 2020-08-01
Forwarded: not-needed

--- json-c-0.15.orig/doc/Doxyfile.in
+++ json-c-0.15/doc/Doxyfile.in
@@ -152,7 +152,7 @@ FULL_PATH_NAMES        = NO
 # will be relative from the directory where doxygen is started.
 # This tag requires that the tag FULL_PATH_NAMES is set to YES.
 
-STRIP_FROM_PATH        =
+STRIP_FROM_PATH        = @CMAKE_SOURCE_DIR@
 
 # The STRIP_FROM_INC_PATH tag can be used to strip a user-defined part of the
 # path mentioned in the documentation of a class, which tells the reader which
